### PR TITLE
Add reactive-elements@0.6.7 w/ npm auto-update

### PR DESCRIPTION
**Pull request by clairetsai818**
Pull request for issue: #10667
Related issue(s): # #

@cdnjs/intern3 Please help me review this PR, thanks!

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/PixelsCommander/ReactiveElements
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/reactive-elements
 * License and its reference: [MIT](https://github.com/PixelsCommander/ReactiveElements/blob/master/README.md#License)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 24
   - Count of stars: 559
   - Count of forks: 31
 * NPM download stats (optional):
   - Downloads in the last day: 10
   - Downloads in the last week: 191
   - Downloads in the last month: 558

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/reactive-elements/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "keywords": [
+    "react",
+    "web-components",
+    "custom elements",
+    "reactjs"
+  ],
+  "author": {
+    "name": "Denis Radin aka PixelsCommander",
+    "email": "denis.radin@gmail.com",
+    "url": "http://pixelscommander.com/"
```
Comment (hare1039) [inline ajax/libs/reactive-elements/package.json]: How about https

File: ajax/libs/reactive-elements/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  },
+  "license": "MIT",
+  "npmName": "reactive-elements",
```
Comment (hare1039) [inline ajax/libs/reactive-elements/package.json]: Here use npm auto-update, please correct the commit message.

File: ajax/libs/reactive-elements/package.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "reactive-elements",
+  "version": "0.6.7",
+  "description": "Allows to use React.js component as custom element",
+  "filename": "reactive-elements.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/PixelsCommander/ReactiveElements.git"
+  },
+  "keywords": [
```
Comment (hare1039) [inline ajax/libs/reactive-elements/package.json]: I think we can use `https://pixelscommander.com/polygon/reactive-elements/example/` as homepage.

**Comment by sufuf3** [comment]
ping @extend1994

**Comment by clairetsai818** [comment]
@PeterDaveHello Please help me review this PR, thanks!

**Comment by extend1994** [comment]
@sufuf3 As what Peter said, I think after two reviewers who approve the pull request are enough, we can request review from our mentor?  Except when two of us approved but one still requested some changes. Just let you know in this case if I forgot to review is okay.

File: ajax/libs/reactive-elements/package.json (hunk 3 above)
Comment (PeterDaveHello) [inline ajax/libs/reactive-elements/package.json]: @cdnjs/intern3 can you guys "discuss" first? It's a shame that somebody says something that another one just do it without any discuss here as you all are still in a early learning phase.

File: ajax/libs/reactive-elements/package.json (hunk 1 above)
Comment (PeterDaveHello) [inline ajax/libs/reactive-elements/package.json]: Why @clairetsai818 just updated the PR without any response comment to the review? Is this kind of "Teacher says"?

File: ajax/libs/reactive-elements/package.json (hunk 3 above)
Comment (extend1994) [inline ajax/libs/reactive-elements/package.json]: I missed this, sorry!
In my opinion, the simple demo page can not be a homepage. The homepage should involve more complete information like documentation, library installation steps and so on.

Comment (PeterDaveHello) [inline ajax/libs/reactive-elements/package.json]: The homepage should at least have download or repo links.

**Comment by clairetsai818** [comment]
@PeterDaveHello I removed the `homepage` field, please help me review this, thanks!

**Comment by PeterDaveHello** [comment]
Reply https://github.com/cdnjs/cdnjs/pull/10729#discussion_r103837438 here first.

**Comment by clairetsai818** [comment]
@PeterDaveHello sorry that I didn't notice that if this website is suitable to be the homepage.

**Comment by PeterDaveHello** [comment]
So please be careful and please "discuss" @clairetsai818

**Pull request merged**